pkg/apimachinery/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/grafana/grafana/pkg/apimachinery
 go 1.23.1
 
 require (
-	github.com/grafana/authlib v0.0.0-20250108202437-7a039176d884 // @grafana/identity-access-team
-	github.com/grafana/authlib/claims v0.0.0-20241202085737-df90af04f335 // @grafana/identity-access-team
+	github.com/grafana/authlib v0.0.0-20250120092408-d9d7ace6ab03 // @grafana/identity-access-team
```

Comment:
^^^ still needed 😢 

We should be able to remove the `requester` type soon, and can then remove this dependency